scripts/validate-dashboards.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+# List of files to exclude
+excludes=( "values.schema.json" "home.json" )
+exculde_string=$(IFS="|"; echo "${excludes[*]}")
+
+# Find valid team names
+github_repo="https://github.com/giantswarm/github.git"
+github_repo_directory="$(mktemp -d -t validate-dashboards.XXXXXXXXXX)"
+trap 'rm -rf "$github_repo_directory"' EXIT
+git clone --depth 1 "$github_repo" "$github_repo_directory"
+pushd "$github_repo_directory" > /dev/null || exit 1
+mapfile teams < <(find repositories -name '*.yaml' -printf '%f\n' | cut -d. -f1)
+popd > /dev/null || exit 1
```

Comment:
Why this pushd/popd thing instead on a `cd`?
This code probably deserves some comments.

Reply:
I would have used this:
```suggestion
cd "$github_repo_directory" > /dev/null || exit 1
mapfile teams < <(find repositories -name '*.yaml' -printf '%f\n' | cut -d. -f1)
cd - > /dev/null || exit 1
```